src/sc_handle.rs

```diff
@@ -1,15 +1,15 @@
-use windows_sys::Win32::{Security, System::Services};
+use windows_sys::Win32::System::Services;
 
 /// A handle holder that wraps a low level [`Security::SC_HANDLE`].
```

Comment:
Please don't forget to update the docs accordingly. Please make sure that `cargo doc` builds without warnings or errors! I have been meaning to get this check into the CI, but have not done so for this repo yet.

Reply:
Ping ^

Reply:
done.